src/main/java/frc/robot/hardware/digitalinput/channeled/ChanneledDigitalInput.java

```diff
@@ -11,14 +11,14 @@ public class ChanneledDigitalInput implements IDigitalInput {
 	private final Debouncer debouncer;
 	private final boolean inverted;
 
-	public ChanneledDigitalInput(int channel, double debounceTime, boolean inverted) {
+	public ChanneledDigitalInput(int channel, Debouncer debouncer, boolean inverted) {
 		this.digitalInput = new DigitalInput(channel);
```

Comment:
DI for this too

Reply:
Wdym?

Reply:
for what?

Reply:
The digital input
that's literally all there is in this line 💀

Reply:
How can u further di it?
It already gets a debouncer

Reply:
If I see the word "new" in the constructor you can further di it😘

Reply:
That destroys the purpose of the class then

Reply:
This class is not for building a digital input instance, it's for debouncing logging and getting its value

Reply:
Yeah OK I get it but what will you call it? It's not channeld..

Reply:
Why not? It still is a digital input that is connected to the roborio in a channel regardless of the constructor getting the channel number or not

Reply:
@Nitay4